### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by konstTSK** [mobinjavari/v2ray-api-php#35] [General]
Hi , i find cricical problem

The API returns an incorrect string; if I generate code or paste it from a buffer, the connection does not occur.
The line is missing some parameters.
Compare the lines and you will understand what I'm talking about

![изображение](https://github.com/mobinjavari/v2ray-api-php/assets/33707789/a43f5409-948c-4970-a5a5-84d041a27cdf)

![изображение](https://github.com/mobinjavari/v2ray-api-php/assets/33707789/5b327a5f-250e-4469-ba18-348bbb425777)

How can I get the parameters?

pbk=
security=
sid=

**Comment by konstTSK** [comment]
UPD:  if  update user ( disable )  and  after update ( enable )
user not conected  if until you reboot  xray.   

maybe can be reloaded via API

**Comment by mobinjavari** [comment]
Hello, from the .xuiConfig.json file, you can edit the index of a url array inside vless and change the constant values

The current api provides these facilities, and you can send a message from Telegram for ordering or additional facilities and features

```json
{
  "url": {
    "scheme": "vless",
    "host": "%ADDRESS%",
    "port": "%PORT%",
    "user": "%USER%",
    "query": "type=%TRANSMISSION%&host=%HEADER%&security=none&headerType=http&path=/",
    "fragment": "%EMAIL%"
  }
}
```

edit query 👆
